### ISSUE TITLE
unable to update @inertiajs/inertia through npm

### ISSUE THREAD

**Issue by steven-lury** [inertiajs/inertia#313]
Due to some bug I have to update `@inertiajs`. When I run `npm update @inertiajs/inertia` nothing happened. I think sth related to `package-lock.json`

in package.json:

`"@inertiajs/inertia" : "^0.1.9", 
"@inertiajs/inertia-vue" : "^0.1.4"`
any help

**Comment by claudiodekker** [comment]
Hi,

This is because the dependencies in your `package.json` are 'fighting' each other and are preventing either to be upgraded:

![Screenshot 2020-11-04 at 09 55 11](https://user-images.githubusercontent.com/1752195/98090054-d6e7da00-1e83-11eb-8376-c3163fa7fc13.png)

The best thing to do instead, is to manually update both package's version constraints in your `package.json` [based on the release versions you're trying to update to (see here)](https://github.com/inertiajs/inertia/releases), and then update the dependencies via `npm update`. 

You'll probably see a warning at first, but if you look at the `package-lock.json` file you'll see that it probably upgraded just fine, and if you re-run the `npm update` command again directly after you'll see the warnings are gone:

![Screenshot 2020-11-04 at 10 00 50](https://user-images.githubusercontent.com/1752195/98090675-a18fbc00-1e84-11eb-90b3-a0737638f8c8.png)

Hope this helps!

**Comment by steven-lury** [comment]
@claudiodekker I did it manually. In package.json upgrade @inertiajs/inertia from 0.1.9 to 0.6.0 and also @inertiajs/inertia-vue from  0.1.4 to 0.2.4 and run `npm update` . Unfortunately nothing happened.

**Comment by claudiodekker** [comment]
Hi, this is due to that `@inertiajs/inertia-vue` has a dependency on either `@inertiajs/inertia` version `^0.2.0` or `^0.3.0`. This 'error' is viewable by removing the `node_modules` folder, as well as the `package-lock.json` file, followed by running `npm install`.

[Because of certain NPM behaviour](https://docs.npmjs.com/cli/v6/using-npm/semver#caret-ranges-123-025-004), pre-1.0 is treated slightly differently. For example, if the requirement is set to `^0.1.2`, npm will only grab the patch level updates, and won't update to `0.2.0` or higher versions.

Long story short, **if you want to only update to `@inertiajs/inertia-vue` version `^0.2.4`, your `@inertiajs/inertia` version cannot be above `^0.3.0`**, although I recommend to directly update to the latest versions of both.

![Screenshot 2020-11-04 at 11 17 39](https://user-images.githubusercontent.com/1752195/98099028-5af38f00-1e8f-11eb-824a-ec886fc3a9d4.png)
